README.md

```diff
@@ -47,6 +47,14 @@ docker:
     - "/path/to/another-image-name.tar"
 ```
 
+### Run a Command
+device-init can execute a list of commands on boot. e.g.:
+```yaml
+runcmd:
+  - "apt-get install package-name"
```

Comment:
In this case yaml in unmarshalled in go code and shell command is run also from go code. These issues can happen when running in console, i.e.:

```bash
# Standard command execution
14:52 $ ls -la | grep test
-rw-rw-r-- 1 floreks floreks  895 lut 22 14:52 test.go

# Using 'sh -c' without quotng command argument
14:52 $ sh -c ls -la | grep test
test.go

# Using 'sh -c' with quoting command argument
14:53 $ sh -c "ls -la | grep test"
-rw-rw-r-- 1 floreks floreks  895 lut 22 14:52 test.go
```

However when we use go code to execute these commands we do not rely on standard console behavior. Check out this [go example](https://play.golang.org/p/KY_4S9CF8_) (it won't run in playground because of additional import).

Output:
```
Quoted command: ls -la | grep test
Unquoted command: ls -la | grep test

--------------------------------
Quoted command out:
-rw-rw-r-- 1 floreks floreks  895 lut 22 14:52 test.go

--------------------------------
Unquoted command out:
-rw-rw-r-- 1 floreks floreks  895 lut 22 14:52 test.go
```

As you can see in both cases array object is converted into a single string object and quotes are dropped. 

`config.GetStringSlice` method is part of viper package that uses same yaml package for unmarshalling. In this case there is no risk related to unquoted commands.